src/Controller/Component/CsrfComponent.php

```diff
@@ -120,12 +121,14 @@ public function implementedEvents()
      */
     protected function _setCookie(Request $request, Response $response)
     {
+        $expires = new Time($this->_config['expiry']);
```

Comment:
Didn't crawl through the code, but CookieComponent::_write() uses the same way to set the unix timestamp as string for "expire" on the Response::cookie().

So I assume we're fine.